### ISSUE TITLE
package not found.

### ISSUE THREAD

**Issue by sp1rs** [Adven27/grpc-wiremock#5]
Suddenly I am getting this error 
```
wiremock_1  | Task ':generateJava' is not up-to-date because:
wiremock_1  |   No history is available.
wiremock_1  | Found :generateJava: 
wiremock_1  | Parsed package = [grpc.health.v1] and service = [Health]
wiremock_1  | Found :generateJava: 
.
.
.
wiremock_1                | Compiling with JDK Java compiler API.
wiremock_1                | /usr/src/java-code/src/main/java/io/adven/grpc/wiremock/Translator.java:12: error: package grpc.health.v1 does not exist
wiremock_1                | import grpc.health.v1.HealthGrpc;
```

Did any new change went live?

**Comment by Adven27** [comment]
Nope. Instructions from README still works.

**Comment by sp1rs** [comment]
I am getting this error if I have a different package and the same service name.
```
**wiremock_1  | Parsed package = [supplier_x.supplier.v2] and service = [SupplierService]**
wiremock_1  | Found :generateJava: 
wiremock_1  | Parsed package = [supplier_x.v1.productpb] and service = [ProductService]
wiremock_1  | Found :generateJava: 
**wiremock_1  | Parsed package = [supplier_x.v1.supplierpb] and service = [SupplierService]**
wiremock_1  | Found :generateJava: 

```

If I changed the service name of one of the packages from `SupplierService` to `SupplierService2` it works.

**Comment by Adven27** [comment]
@sp1rs, the issue should be fixed by #7, please check
